### DISCUSSION TITLE
Does zookeeper necessary for single node clickhouse deployment?

### DISCUSSION THREAD

**Discussion by JobberRT** [ClickHouse/ClickHouse#50338] [Q&A] [answered]
I know `ZooKeeper` is used to manage multiple node, but if there's only one clickhouse instance, is zookeeper still a must need?

I read the official doc and see no disable option for zookeeper.

**Comment by DanRoscigno** [comment]
When you install ClickHouse Clickhouse Keeper is disabled by default; unless you enable it in the config it is not listening.

**Comment by den-crane** [comment, marked as answer]
Zookeeper is not necessary for single node clickhouse deployment unless you use Replicated tables. 
You may need Replicated tables with the single node installation if you consider to grow your setup to several nodes in the future.
